tests/test_validator.py

```diff
@@ -330,6 +330,10 @@ def test_too_many_arguments(self):
             with self.assertRaises(expected_exception=TooManyArguments):
                 client.get('/kwargs', data=json.dumps(data), content_type='application/json')
 
+            # test backward compatibility
+            with self.assertRaises(expected_exception=TooMuchArguments):
+                client.get('/kwargs', data=json.dumps(data), content_type='application/json')
+
```

Comment:
@d-ganchar If someone handled the old `TooMuchArguments` exception he / she will also catch the new `TooManyArguments` exception without changing any code lines.